base/mmcif_ma-core.dic

```diff
@@ -828,6 +1083,147 @@ save__ma_template_non_poly.details
     _item_type.code               text
      save_
 
+##########################
+## MA_TEMPLATE_BRANCHED ##
+##########################
+
+save_ma_template_branched
+    _category.description
+;              Data items in the MA_TEMPLATE_BRANCHED category record details about
+               the structural templates for branched polymers used in the 
+               homology/comparative modeling.
+;
+    _category.id                  ma_template_branched
+    _category.mandatory_code      no
+    _category_key.name            '_ma_template_branched.ordinal_id'
+    loop_
+    _category_group.id            'inclusive_group'
+                                  'ma_group'
+    loop_
+    _category_examples.detail
+    _category_examples.case
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+;
+    Example 1 -
+;
```

Comment:
I assembled a nice little example, based on a modelling project I use for SMNG ModelCIF unit tests. Source is PDB entry [3axh](https://www.rcsb.org/structure/3AXH),  entity 2:

```    _ma_template_branched.ordinal_id       1
    _ma_template_branched.template_id      2
    _ma_template_branched.name             alpha-D-glucopyranose-(1-6)-alpha-D-glucopyranose
    _ma_template_branched.descriptor_type  'Glycam Condensed Sequence'
    _ma_template_branched.descriptor_value DGlcpa1-6DGlcpa1-ROH
    _ma_template_branched.type             oligosaccharide
```

Values are copied from `_pdbx_entity_branch_descriptor` beside `_ma_template_branched.type`, which is from `_pdbx_entity_branch.type`. The description could also mention that the table can be filled from pdbx mmCIF table `_pdbx_entity_branch_descriptor`, `_pdbx_entity_branch`.